chore(website): Monthly Product updates 3 (#6989)
cloudquery/cloudquery: website/pages/blog/product-updates-3.md

---
title: "CloudQuery Product Updates #3"
tag: product
date: 2023/01/23
description: >-
  Monthly updates on CloudQuery product and roadmap.
author: yevgenypats
---

import { BlogHeader } from "../../components/BlogHeader"

<BlogHeader/>

Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here! This month was packed!

## SDK Updates

> - We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.

Reviewer: ```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allows you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

> - We now support [incremental syncing](https://www.cloudquery.io/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.

Reviewer 2: ```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

> - We now support [incremental syncing](https://www.cloudquery.io/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.

Reviewer 2: ```suggestion
- We now support [incremental syncing](/docs/core-concepts/syncs#incremental-table-syncs) in alpha! This allow you to sync only new data from your sources. This is a huge performance improvement for large tables.
```

## Destinations

### New Destinations

We've added 7 new destinations to CQ!!

* [MongoDB](https://www.cloudquery.io/docs/plugins/destinations/mongodb/overview)
* [Neo4j](https://www.cloudquery.io/docs/plugins/destinations/neo4j/overview)

> * [Kafka](https://www.cloudquery.io/docs/plugins/destinations/kafka/overview)

Reviewer 2: ```suggestion
* [Kafka](/docs/plugins/destinations/kafka/overview)
```

* [MSSQL](https://www.cloudquery.io/docs/plugins/destinations/mssql/overview)

> * [GCS](https://www.cloudquery.io/docs/plugins/destinations/gcs/overview)

Reviewer 2: ```suggestion
* [GCS](/docs/plugins/destinations/gcs/overview)
```

> * [S3](https://www.cloudquery.io/docs/plugins/destinations/s3/overview)

Reviewer 2: ```suggestion
* [S3](/docs/plugins/destinations/s3/overview)
```

> * [File](https://www.cloudquery.io/docs/plugins/destinations/file/overview)

Reviewer 2: ```suggestion
* [File](/docs/plugins/destinations/file/overview)
```

> * [Azure Blob Storage](https://www.cloudquery.io/docs/plugins/destinations/azblob/overview)

Reviewer 2: ```suggestion
* [Azure Blob Storage](/docs/plugins/destinations/azblob/overview)
```

## Sources

### New Sources

> We've added 7 new source plugins to CQ to cover the long tail of infrastructure apps!

Reviewer 2: ```suggestion
We've added 8 new source plugins to CQ to cover the long tail of infrastructure apps!
```

> * [Slack](https://www.cloudquery.io/docs/plugins/sources/slack/overview)

Reviewer 2: ```suggestion
* [Slack](/docs/plugins/sources/slack/overview)
```

> * [PagerDuty](https://www.cloudquery.io/docs/plugins/sources/pagerduty/overview)

Reviewer 2: ```suggestion
* [PagerDuty](/docs/plugins/sources/pagerduty/overview)
```

> * [Alibaba Cloud](https://www.cloudquery.io/docs/plugins/sources/alicloud/overview)

Reviewer 2: ```suggestion
* [Alibaba Cloud](/docs/plugins/sources/alicloud/overview)
```

> * [Oracle](https://www.cloudquery.io/docs/plugins/sources/oracle/overview)

Reviewer 2: ```suggestion
* [Oracle](/docs/plugins/sources/oracle/overview)
```

* [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)

> * [Hacker News](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)

Reviewer: ```suggestion
* [Hacker News](/docs/plugins/sources/hackernews/overview)
```

> * [GitLab](https://www.cloudquery.io/docs/plugins/sources/gitlab/overview)

Reviewer 2: ```suggestion
* [GitLab](/docs/plugins/sources/gitlab/overview)
```

> * [HackNews](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)

Reviewer 2: ```suggestion
* [Hacker News](https://www.cloudquery.io/docs/plugins/sources/hackernews/overview)
```

> * [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview)

Reviewer 2: ```suggestion
* [Shopify](/docs/plugins/sources/shopify/overview)
```

> * [Stripe](https://www.cloudquery.io/docs/plugins/sources/stripe/overview)

Reviewer 2: ```suggestion
* [Stripe](/docs/plugins/sources/stripe/overview)
```

> Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md).

Reviewer 2: ```suggestion
Numerous updates to [AWS](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/aws/CHANGELOG.md), [GCP](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/gcp/CHANGELOG.md) and [Azure](https://github.com/cloudquery/cloudquery/blob/40ff6b263b4ad0e96b00495731ed8f4247f87b52/plugins/source/azure/CHANGELOG.md).
```

We should pin these files to a specific commit so their content reflects the time the blog is published (you can do this by clicking `y` on your keyboard when you have a file open)

## Other notable blogs and use-cases

### Engineering

> - [Reducing Alert Fatigue with the PagerDuty Plugin](https://www.cloudquery.io/blog/announcing-pagerduty-plugin).

Reviewer 2: ```suggestion
- [Reducing Alert Fatigue with the PagerDuty Plugin](/blog/announcing-pagerduty-plugin).
```

### Security

> - [How to use CloudQuery for Attack Surface Management and Graph Visualization](https://www.cloudquery.io/how-to-guides/attack-surface-management-with-graph)

Reviewer 2: ```suggestion
- [How to use CloudQuery for Attack Surface Management and Graph Visualization](/how-to-guides/attack-surface-management-with-graph)
```

### FinOps

> - [https://www.cloudquery.io/blog/azure-cost-optimization-with-cloudquery](https://www.cloudquery.io/blog/azure-cost-optimization-with-cloudquery)

Reviewer 2: ```suggestion
- [Correlating Costs to Resources for Azure Cost Optimization with CloudQuery](/blog/azure-cost-optimization-with-cloudquery)
```
